examples/ruby/README.md

```diff
@@ -1,3 +1,7 @@
+**API `v1` is deprecated**
+
+***
+
```

Comment:
@seven1m It don't think https://github.com/planningcenter/pco_api_ruby has been updated for the new format has it?  Readme still shows top level attributes in the data